shared/src/main/scala/squants/package.scala

```diff
@@ -125,5 +125,16 @@ package object squants {
    * @param d Double number to be formatted
    * @return
    */
-  private[squants] def crossFormat(d: Double): String = if (d.toLong == d) { "%.1f".format(d) } else { d.toString }
+  private[squants] def crossFormat(d: Double): String = {
```

Comment:
On `scala.js` we can put different implementations of a class for each platform under the `js` and `jvm` folders. Can we do the same and put the native specific code is a separate location?